### DISCUSSION TITLE
Problem with template: Massively background image

### DISCUSSION THREAD

**Discussion by Gruhler** [GetPublii/Publii#1606] [Help]
Problem with template: Massively, the page background image cannot be integrated. 
Does anyone have a solution or an idea?

**Comment by candidexmedia** [comment]
Can you share a live link and/or screenshot?

**Reply by Gruhler** [reply]
Test all this image:
![grafik](https://github.com/user-attachments/assets/efd5cdd2-bf86-4c0c-a1de-0a33846d684a)
nur gekreisle .-))
![grafik](https://github.com/user-attachments/assets/1f048872-7893-46b5-ad71-b30b4d0fe379)

**Reply by dractyl** [reply]
I have this exact same problem. Uploading times out forever. Website Logo fails in the same way. My Publii language is English, FWIW. Uploading pictures on a post or page works fine.

I am using 0.46.0 (build 16910) on macOS Sonoma 14.5.

**Reply by dractyl** [reply]
I have the same behavior on 0.46.1 (build 16918). I opened up the DevTools and checked the Javascript console, but there were no relevant errors there either (only the TinyMCE deprecation).

**Comment by stormare** [comment]
Hi all, is the issue solved?
I have also issue with massively background setting, in my case **no image is shown** - even though I put there the original theme background image. I tried all kinds of formats, but nothing works.

<img width="504" height="221" alt="Screenshot 2025-12-09 215758" src="https://github.com/user-attachments/assets/7e34ea9c-7fe9-4126-9122-76baebc81a48" />

Can somebody help me with this?

**Reply by michaelfluegge** [reply]
this might be dependent from your changes on the theme. I suggest to clone the site, reset the theme settings and see what happens then

**Reply by michaelfluegge** [reply]
disable Plugins, Custom CSS and/or Custom HTML might also help for testing

**Reply by stormare** [reply]
Thank you for the answer!
I created new website, no plugins, no settings, just activated the Massively theme and added background image.
This is the result, no image shown:

<img width="928" height="693" alt="image" src="https://github.com/user-attachments/assets/91378a87-2028-4c69-92a7-0c7043a43d50" />

**Comment by michaelfluegge** [comment]
Ok, I did the same with Publii 0.47.3 (build 17367) – Theme Massively v.2.2.0.0 under MacOS: 

<img width="1296" height="839" alt="Bildschirmfoto 2025-12-10 um 23 32 46" src="https://github.com/user-attachments/assets/08e2e19e-a818-4440-9430-a12bc7971c99" />

**Reply by michaelfluegge** [reply]
are you using the most current version of Publii?

**Reply by stormare** [reply]
Yes, I downloaded it 2 days ago, but it is for Windows.

**Reply by stormare** [reply]
I see the image in the CSS when inspecting the website:

<img width="760" height="285" alt="image" src="https://github.com/user-attachments/assets/01c34632-4bf3-4a28-bd2d-07c067f91820" />

But probably it is under the overlay.

**Reply by michaelfluegge** [reply]
I checked it again with Safari and Firefox, and both work fine.
I am not Pro enough to say whether you have an OS, or a browser related problem.

**Reply by michaelfluegge** [reply]
maybe @candidexmedia has an idea?

**Reply by TaifunTux** [reply]
I’m having the same problem. Can you find out which file contains the reference to the background image?

**Comment by bobmitro** [comment]
@stormare When you preview the website, open the browser's developer tools (F12), check the Console tab, and share the output here, or best share a link to the live site. That'll help us spot what's going on.

**Reply by stormare** [reply]
Thanks for your reply.
I checked and the issue is, that the images are in wrong folder.

<img width="951" height="389" alt="image" src="https://github.com/user-attachments/assets/9fcf7511-ee25-43c5-85b0-fc37704ffd75" />

They should be in the "images" folder but instead the background was created in the media/website folder:

<img width="371" height="179" alt="image" src="https://github.com/user-attachments/assets/ef58e61b-811d-4b0d-965d-3f1af08abf1c" />

**Reply by TaifunTux** [reply]
@stormare, does this solve the problem? I can see that, on your system, the ‘image’ folder is at the same level as the ‘assets’ folder. On my system, ‘image’ is a subfolder of ‘assets’.

**Comment by stormare** [comment]
No, it did not work, so I switched to different website tool :(

**Reply by TaifunTux** [reply]
Switch to which other tool?

**Comment by stormare** [comment]
Sorry, cannot help here, I already deleted everything from Publii as I did not solve the issue for weeks.

-------- Original Message --------

**Comment by TaifunTux** [comment]
I was able to solve the problem. It was the name of the root folder. An ampersand (&) was used in it, and that was the culprit.
